lightning/src/ln/channelmanager.rs

```diff
@@ -4341,7 +4346,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 				if !self.default_configuration.manually_accept_inbound_channels {
 					channel_state.pending_msg_events.push(events::MessageSendEvent::SendAcceptChannel {
 						node_id: counterparty_node_id.clone(),
-						msg: channel.accept_inbound_channel(),
+						msg: channel.accept_inbound_channel(0),
```

Comment:
Oops, this now overwrites the user id for outbound channels - can either make the `channel.rs` function an `Option` again or re-set it by fetching the channel's ID here.

Reply:
It's not clear for me how this happens as `accept_inbound_channel` doesn't allow outbound channels modifications https://github.com/lightningdevkit/rust-lightning/blob/edd4babb1cf886654259ecad8594661e829a0680/lightning/src/ln/channel.rs#L4748-L4751

Reply:
Oh! I'm sorry, yes. So this comment is in `internal_open_channel`, not `accept_inbound_channel`, but I clearly wasn't thinking, that is also for inbound channels so no problem.